ERROR - 2023-10-18 14:56:28 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(29): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 14:56:28 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`) VALUES (:largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(29): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 14:56:42 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(29): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 14:56:42 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`) VALUES (:largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(29): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 14:59:38 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(29): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 14:59:38 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`) VALUES (:largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(29): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 14:59:45 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(29): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 14:59:45 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`) VALUES (:largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(29): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:07:48 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:07:48 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`) VALUES (:largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(31): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:07:52 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:07:52 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`) VALUES (:largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(31): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:16:39 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:16:39 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`) VALUES (:largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(31): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:16:45 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:16:45 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`falda`, CONSTRAINT `fk_falda_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`) VALUES (:largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(31): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:18:22 --> mysqli_sql_exception: Column 'cliente_id' cannot be null in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(26): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:18:22 --> Column 'cliente_id' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`, `cliente_id`) VALUES (:largo:, :cintura:, :cadera:, :cliente_id:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(26): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:18:26 --> mysqli_sql_exception: Column 'cliente_id' cannot be null in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(26): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:18:26 --> Column 'cliente_id' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`, `cliente_id`) VALUES (:largo:, :cintura:, :cadera:, :cliente_id:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(26): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:20:45 --> mysqli_sql_exception: Column 'cliente_id' cannot be null in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(40): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:20:45 --> Column 'cliente_id' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`, `cliente_id`) VALUES (:largo:, :cintura:, :cadera:, :cliente_id:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(40): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:20:52 --> mysqli_sql_exception: Column 'cliente_id' cannot be null in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(40): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:20:52 --> Column 'cliente_id' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`, `cliente_id`) VALUES (:largo:, :cintura:, :cadera:, :cliente_id:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(40): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 15:23:06 --> Too few arguments to function App\Controllers\Faldas::guardar(), 0 passed in C:\xampp\htdocs\code4\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\Faldas.php on line 23.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 15:23:35 --> Too few arguments to function App\Controllers\Faldas::guardar(), 0 passed in C:\xampp\htdocs\code4\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\Faldas.php on line 23.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:24:32 --> mysqli_sql_exception: Column 'cliente_id' cannot be null in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(39): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:24:32 --> Column 'cliente_id' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`, `cliente_id`) VALUES (:largo:, :cintura:, :cadera:, :cliente_id:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(39): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 15:25:04 --> Too few arguments to function App\Controllers\Faldas::falda(), 0 passed in C:\xampp\htdocs\code4\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\Faldas.php on line 9.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->falda()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 15:25:10 --> Too few arguments to function App\Controllers\Faldas::falda(), 0 passed in C:\xampp\htdocs\code4\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\Faldas.php on line 9.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->falda()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 15:25:13 --> Too few arguments to function App\Controllers\Faldas::falda(), 0 passed in C:\xampp\htdocs\code4\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\Faldas.php on line 9.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->falda()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 15:25:15 --> Too few arguments to function App\Controllers\Faldas::falda(), 0 passed in C:\xampp\htdocs\code4\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\Faldas.php on line 9.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->falda()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 15:25:16 --> Too few arguments to function App\Controllers\Faldas::falda(), 0 passed in C:\xampp\htdocs\code4\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\Faldas.php on line 9.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->falda()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 15:25:22 --> Too few arguments to function App\Controllers\Faldas::falda(), 0 passed in C:\xampp\htdocs\code4\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\Faldas.php on line 9.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->falda()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 15:25:25 --> Too few arguments to function App\Controllers\Faldas::falda(), 0 passed in C:\xampp\htdocs\code4\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\Faldas.php on line 9.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->falda()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 15:26:02 --> Too few arguments to function App\Controllers\Faldas::guardar(), 0 passed in C:\xampp\htdocs\code4\system\CodeIgniter.php on line 919 and exactly 1 expected
in APPPATH\Controllers\Faldas.php on line 23.
 1 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 2 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 3 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:36:37 --> mysqli_sql_exception: Column 'cliente_id' cannot be null in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:36:37 --> Column 'cliente_id' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`largo`, `cintura`, `cadera`, `cliente_id`) VALUES (:largo:, :cintura:, :cadera:, :cliente_id:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(30): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 15:44:33 --> mysqli_sql_exception: Duplicate entry '71' for key 'falda.PRIMARY' in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 15:44:33 --> Duplicate entry '71' for key 'falda.PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`cliente_id`, `largo`, `cintura`, `cadera`) VALUES (:cliente_id:, :largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(30): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:04:15 --> mysqli_sql_exception: Duplicate entry '72' for key 'falda.PRIMARY' in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:04:15 --> Duplicate entry '72' for key 'falda.PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`cliente_id`, `largo`, `cintura`, `cadera`) VALUES (:cliente_id:, :largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(30): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:04:25 --> mysqli_sql_exception: Duplicate entry '72' for key 'falda.PRIMARY' in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(30): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:04:25 --> Duplicate entry '72' for key 'falda.PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`cliente_id`, `largo`, `cintura`, `cadera`) VALUES (:cliente_id:, :largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(30): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:22:40 --> mysqli_sql_exception: Duplicate entry '71' for key 'falda.PRIMARY' in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:22:40 --> Duplicate entry '71' for key 'falda.PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`cliente_id`, `largo`, `cintura`, `cadera`) VALUES (:cliente_id:, :largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:26:31 --> mysqli_sql_exception: Duplicate entry '72' for key 'falda.PRIMARY' in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `fa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `fa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `fa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `fa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Faldas.php(45): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:26:31 --> Duplicate entry '72' for key 'falda.PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `falda` (`cliente_id`, `largo`, `cintura`, `cadera`) VALUES (:cliente_id:, :largo:, :cintura:, :cadera:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Faldas.php(45): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Faldas->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Faldas))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:49:14 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`pantalon`, CONSTRAINT `fk_pantalon_cliente` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Pantalons.php(33): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Pantalons->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pantalons))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:49:14 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`pantalon`, CONSTRAINT `fk_pantalon_cliente` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pantalon` (`largo`, `entrepierna`, `cintura`, `cadera`, `pierna`, `rodilla`) VALUES (:largo:, :entrepierna:, :cintura:, :cadera:, :pierna:, :rodilla:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Pantalons.php(33): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Pantalons->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pantalons))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:49:49 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`pantalon`, CONSTRAINT `fk_pantalon_cliente` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pa...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pa...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pa...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pa...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\Pantalons.php(33): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\Pantalons->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pantalons))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:49:49 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`pantalon`, CONSTRAINT `fk_pantalon_cliente` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pantalon` (`cliente_id`, `largo`, `entrepierna`, `cintura`, `cadera`, `pierna`, `rodilla`) VALUES (:cliente_id:, :largo:, :entrepierna:, :cintura:, :cadera:, :pierna:, :rodilla:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Pantalons.php(33): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Pantalons->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pantalons))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:51:42 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`traje_femenino`, CONSTRAINT `fk_traje_femenino_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tr...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\TrajeFemeninos.php(36): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\TrajeFemeninos->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TrajeFemeninos))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:51:42 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`traje_femenino`, CONSTRAINT `fk_traje_femenino_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `traje_femenino` (`cliente_id`, `talle`, `largo`, `hombro`, `ancho`, `pecho`, `cintura`, `cadera`, `largoManga`) VALUES (:cliente_id:, :talle:, :largo:, :hombro:, :ancho:, :pecho:, :cintura:, :cadera:, :largoManga:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\TrajeFemeninos.php(36): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\TrajeFemeninos->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TrajeFemeninos))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:51:49 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`traje_femenino`, CONSTRAINT `fk_traje_femenino_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tr...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\TrajeFemeninos.php(36): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\TrajeFemeninos->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TrajeFemeninos))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:51:49 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`traje_femenino`, CONSTRAINT `fk_traje_femenino_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `traje_femenino` (`cliente_id`, `talle`, `largo`, `hombro`, `ancho`, `pecho`, `cintura`, `cadera`, `largoManga`) VALUES (:cliente_id:, :talle:, :largo:, :hombro:, :ancho:, :pecho:, :cintura:, :cadera:, :largoManga:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\TrajeFemeninos.php(36): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\TrajeFemeninos->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TrajeFemeninos))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:52:46 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`traje_femenino`, CONSTRAINT `fk_traje_femenino_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tr...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\TrajeFemeninos.php(36): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\TrajeFemeninos->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TrajeFemeninos))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:52:46 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`traje_femenino`, CONSTRAINT `fk_traje_femenino_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `traje_femenino` (`cliente_id`, `talle`, `largo`, `hombro`, `ancho`, `pecho`, `cintura`, `cadera`, `largoManga`) VALUES (:cliente_id:, :talle:, :largo:, :hombro:, :ancho:, :pecho:, :cintura:, :cadera:, :largoManga:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\TrajeFemeninos.php(36): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\TrajeFemeninos->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TrajeFemeninos))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:53:22 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`traje_masculino`, CONSTRAINT `fk_traje_masculino_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tr...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\TrajeMasculinos.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\TrajeMasculinos->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TrajeMasculinos))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:53:22 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`traje_masculino`, CONSTRAINT `fk_traje_masculino_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `traje_masculino` (`cliente_id`, `talle`, `largo`, `hombro`, `ancho`, `pecho`, `estomago`, `largoManga`) VALUES (:cliente_id:, :talle:, :largo:, :hombro:, :ancho:, :pecho:, :estomago:, :largoManga:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\TrajeMasculinos.php(31): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\TrajeMasculinos->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TrajeMasculinos))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2023-10-18 18:53:26 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`traje_masculino`, CONSTRAINT `fk_traje_masculino_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`)) in C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\code4\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `tr...', 0)
#1 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 C:\xampp\htdocs\code4\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 C:\xampp\htdocs\code4\system\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 C:\xampp\htdocs\code4\system\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\code4\system\BaseModel.php(788): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\code4\system\Model.php(715): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\code4\app\Controllers\TrajeMasculinos.php(31): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\code4\system\CodeIgniter.php(919): App\Controllers\TrajeMasculinos->guardar()
#9 C:\xampp\htdocs\code4\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TrajeMasculinos))
#10 C:\xampp\htdocs\code4\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\code4\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-10-18 18:53:26 --> Cannot add or update a child row: a foreign key constraint fails (`bddsastreria`.`traje_masculino`, CONSTRAINT `fk_traje_masculino_cliente1` FOREIGN KEY (`cliente_id`) REFERENCES `cliente` (`id`))
in SYSTEMPATH\Database\BaseConnection.php on line 646.
 1 SYSTEMPATH\Database\BaseBuilder.php(2308): CodeIgniter\Database\BaseConnection->query('INSERT INTO `traje_masculino` (`cliente_id`, `talle`, `largo`, `hombro`, `ancho`, `pecho`, `estomago`, `largoManga`) VALUES (:cliente_id:, :talle:, :largo:, :hombro:, :ancho:, :pecho:, :estomago:, :largoManga:)', [...], false)
 2 SYSTEMPATH\Model.php(319): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(788): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(715): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\TrajeMasculinos.php(31): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\TrajeMasculinos->guardar()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\TrajeMasculinos))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:01:46 --> Undefined variable $productos
in APPPATH\Views\sastreria\tienda.html on line 89.
 1 APPPATH\Views\sastreria\tienda.html(89): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $productos', 'C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.html', 89)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.html')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda.html', [], true)
 5 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda.html')
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:05:29 --> Invalid file: "sastreria/tienda.html"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('sastreria/tienda.html')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda.html', [], true)
 3 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda.html')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:07:34 --> Invalid file: "sastreria/tienda.html"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('sastreria/tienda.html')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda.html', [], true)
 3 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda.html')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:07:35 --> Invalid file: "sastreria/tienda.html"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('sastreria/tienda.html')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda.html', [], true)
 3 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda.html')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:07:36 --> Invalid file: "sastreria/tienda.html"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('sastreria/tienda.html')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda.html', [], true)
 3 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda.html')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:07:36 --> Invalid file: "sastreria/tienda.html"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('sastreria/tienda.html')
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda.html', [], true)
 3 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda.html')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:12:40 --> Undefined variable $libro
in APPPATH\Views\sastreria\tienda.php on line 105.
 1 APPPATH\Views\sastreria\tienda.php(105): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $libro', 'C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.php', 105)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda', [], true)
 5 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda')
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:13:30 --> Undefined variable $productos
in APPPATH\Views\sastreria\tienda.php on line 89.
 1 APPPATH\Views\sastreria\tienda.php(89): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $productos', 'C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.php', 89)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda', [], true)
 5 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda')
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:39:22 --> Undefined variable $productos
in APPPATH\Views\sastreria\tienda.php on line 91.
 1 APPPATH\Views\sastreria\tienda.php(91): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $productos', 'C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.php', 91)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda', [], true)
 5 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda')
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:40:55 --> syntax error, unexpected token "?"
in APPPATH\Views\sastreria\tienda.php on line 109.
 1 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda', [], true)
 3 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:42:24 --> Undefined variable $libro
in APPPATH\Views\sastreria\tienda.php on line 109.
 1 APPPATH\Views\sastreria\tienda.php(109): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $libro', 'C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.php', 109)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda', [], true)
 5 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda')
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:42:25 --> Undefined variable $libro
in APPPATH\Views\sastreria\tienda.php on line 109.
 1 APPPATH\Views\sastreria\tienda.php(109): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $libro', 'C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.php', 109)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\code4\\app\\Views\\sastreria\\tienda.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda', [], true)
 5 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda')
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 20:53:02 --> Undefined variable $cabecera
in APPPATH\Views\productos\producto.php on line 1.
 1 APPPATH\Views\productos\producto.php(1): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $cabecera', 'C:\\xampp\\htdocs\\code4\\app\\Views\\productos\\producto.php', 1)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\code4\\app\\Views\\productos\\producto.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('productos/producto', [], true)
 5 APPPATH\Controllers\Productos.php(14): view('productos/producto', [...])
 6 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Productos->producto()
 7 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Productos))
 8 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-10-18 21:42:05 --> syntax error, unexpected token "<", expecting "elseif" or "else" or "endif"
in APPPATH\Views\sastreria\tienda.php on line 130.
 1 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1188): CodeIgniter\View\View->render('sastreria/tienda', [], true)
 3 APPPATH\Controllers\Clientes.php(73): view('sastreria/tienda')
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Clientes->tienda()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Clientes))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
